docs/api-reference/loaders/glb-writer.md

```diff
@@ -0,0 +1,39 @@
+# GLBWriter
+
+The `GLBWriter` supports encoding of GLB files.
+
+
+## `GLBBuilder` class
+
+The `GLBWriter` module exposes the `GLBBuilder` class that allows applications to dynamically build up a hybrid JSON/binary GLB file. It exposes the following methods:
+
+
+### constructor
+
+Creates a new `GLBBuilder` instance.
+
+
+### addBuffer(typedArray : TypedArray, accessor : Object) : Number
+
+Adds one binary array intended to be loaded back as a WebGL buffer.
+
+* `typedArray` -
+* `accessor` - {size, type, ...}.
+
+Type is autodeduced from the type of the typed array.
+
+The binary data will be added to the GLB BIN chunk, and glTF `bufferView` and `accessor` fields will be populated.
+
+
+### addImage(typedArray: TypedArray) : Number
+
+Adds an image
+
+The binary image data will be added to the GLB BIN chunk, and glTF `bufferView` and `image` fields will be populated.
+
+
+### encode(json: Object | Array, options : Object) : ArrayBuffer
```

Comment:
what are the options?

One option we need it a **filter** on what arrays are encoded (to selectively encode only large contiguous arrays for example).

Reply:
> One option we need it a filter on what arrays are encoded

Yes, good point.

Let me go through options and docs in the next PR. This PR is getting big.